build-system/tasks/check-types.js

```diff
@@ -204,7 +204,7 @@ async function typeCheck(targetName) {
 
   // If srcGlobs and externGlobs are defined, determine the externs/extraGlobs
   if (srcGlobs.length || externGlobs.length) {
-    opts.externs = externGlobs.flatMap(globby.sync);
```

Comment:
unrelated note: we should go through all our usages of `globby.sync` and switch to async

Reply:
Same with file I/O, we do a looot of `readFileSync` etc iirc